indicator-stickynotes.py

```diff
@@ -256,8 +256,13 @@ def handler(indicator):
     #indicator.nset.notes[current_note].show()
     #indicator.nset.current_note = current_note
 
+def reload_handler(indicator):
+    # reload from data file on SIGUSR2
+    with open(os.path.expanduser(indicator.data_file), encoding="utf-8") as fsock:
+        indicator.nset.merge(fsock.read())
+    install_glib_handler(indicator, signal.SIGUSR2)
```

Comment:
Have you thought about how this behaves in case notes are being edited? Could it cause a crash or unintended overwriting of data? I don't necessarily want to sort it out right now just curious.